ospfd/ospf_vty.c

```diff
@@ -11163,11 +11184,16 @@ DEFUN (show_ip_ospf_route,
 	int idx_vrf = 0;
 	uint8_t use_vrf = 0;
 	bool uj = use_json(argc, argv);
+	bool verbose = false;
 	json_object *json = NULL;
 
 	if (uj)
 		json = json_object_new_object();
 
+	if (argv[argc - (uj == true? 2: 1)]->arg &&
```

Comment:
Convert this function DEFUN to DEFPY and you will be able to avoid this _funky_ logic here and below. See for more examples like (`debug_ospf_packet_cmd` or so).

Reply:
I choose to use `argv_find()` to parse user's input like `show_ip_ospf_gr_helper()` do.